### DISCUSSION TITLE
Raspberry Pi

### DISCUSSION THREAD

**Discussion by gooseisgoose** [3kh0/ext-remover#885] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
anything that i could do with my raspberry pi? my district figured out how to disable everything on here.

**Comment by AshtonDavies** [comment]
I like how this dude pulls up this discussion after I unplug my raspberry pi server. 👀

**Reply by gooseisgoose** [reply]
so is there anything i can do with it. like stuff on this.

**Reply by AshtonDavies** [reply]
Ok but for real, what kind of setup do you currently have?

What I mean is, is it school-owned or something and you need an unblock?

**Reply by gooseisgoose** [reply]
a pi 4 model b (I haven't done much with it yet). and display. just a basic setup.  i have a zero w. idk i'm kidda new to pi and stupid. and i own both. maybe this was a stupid question.

**Reply by gooseisgoose** [reply]
is it possible to run something like kali on pi and display it on my school computer (basically a chromebook ). I pretty fucking stupid.

**Reply by AshtonDavies** [reply]
Yeah you should first of all flash with the Pi imager on another PC. You can just go with the basic Raspberry Pi OS (Raspbian) or I would personally recommend Twister OS for its nice themes. When you have that flashed, you will have Chromium and plenty of other applications to use.

The biggest issue I had back when I used it as a PC was the limited ARM processor, which is slowly supporting more stuff. There's a lot of supported applications to install in Pi Apps.

**Reply by AshtonDavies** [reply]
> is it possible to run something like kali on pi and display it on my school computer (basically a chromebook ). I pretty fucking stupid. 

Kali is supported on ARM. ✅
Then use VNC viewer ig idk.

Or just pull up Mercury Workshop's Anura OS website (https://anura.pro) on your school computer for a Chrome OS-like environment. I think it's down though.

**Reply by gooseisgoose** [reply]
ok thanks.

i'll try your os recommendation to i hate Raspberry Pi OS styles.

**Reply by AshtonDavies** [reply]
> ok thanks.
> 
> i'll try your os recommendation to i hate Raspberry Pi OS looks.
> 

Agreed 👍

**Comment by gooseisgoose** [comment]
another dumb ass question how do I connect it to my school computer so that the school computer displays the raspberry pi.

**Reply by AshtonDavies** [reply]
Are you able to install VNC Viewer on the school computer? Chrome Remote Desktop is more likely available for school devices, but it wouldn't work well for this situation because of limitations and purpose.

**Reply by AshtonDavies** [reply]
I also know of a cool thing that you could do with both Xming and Putty to open apps of your Pi through the terminal of another computer as if it's an application on that computer. You'd have to setup a tunnel to access it though, so probably not a quick off the top option but it's cool.

**Reply by gooseisgoose** [reply]
no, even Chrome remote Desktop is blocked. one last thing maybe there's some way to use some type of cable to plug in the pi to my school computer. if not all good. DO you know some good pi projects.

**Reply by AshtonDavies** [reply]
> no, even Chrome remote Desktop is blocked. one last thing maybe there's some way to use some type of cable to plug in the pi to my school computer. if not all good. DO you know some good pi projects. 

If you mean a Chromebook, no, not really. I think you said you had a Pi Zero, right? If you get a few other things you could build a MintyPi. I haven't done it before but it's a popular project if you want to turn an Altoids container into a gaming system and play it in class. The Pi 4 could run a game server for your friends or be a retro gaming console with RetroPi.

**Reply by gooseisgoose** [reply]
i mean its not exactly a chromebook its a "DELL" its still shit.

and how would i host a game server.

and just a tip don't use p4wn pi at school. it doesn't end well.

**Reply by AshtonDavies** [reply, marked as answer]
I believe there should still be a Minecraft server thing in Pi Apps if you hop on your Twister OS installation. Other games might not be in there. I run it on a plain RpiOS installation for best performance and because I have desktop PCs now. I already know you're not going to do port forwarding crap, so you will need to get the Playit.gg application and link it to a tunnel that you set up on their website. It's all free and it's fun to play with a group of friends. I recommend using Purpur as your server executable if you are going to host a Minecraft server. I don't have experience with other game servers.

**Reply by gooseisgoose** [reply]
ok thanks for ur help.
